src/main/java/uk/nhs/ctp/audit/sqs/AWSAuditSender.java

```diff
@@ -55,6 +56,12 @@ public void sendAudit(AuditSession session) {
                 .withQueueUrl(loggingQueue)
                 .withMessageBody(mapper.writeValueAsString(session));
             sqsClient.sendMessage(request);
+        } catch (AmazonSQSException e) {
+            if (e.getStatusCode() == 413) {
+                log.warn("Audit request exceeded max size SQS can handle", e);
+                //TODO: CDSCT-338 - Should we be auditing/paging audit search audits or excluding things
```

Comment:
The general advice with SQS is if the payload is too large, it should be stored in S3 and sent as a reference instead.

In this case, I think we probably want to exclude auditing other audit-related calls as it's outside the CDS API anyway

Reply:
I'll add that to the ticket